lib/Helper.php

```diff
@@ -156,10 +156,6 @@ public function haveDisabledConfigurations() {
 		$all = $this->getServerConfigurationPrefixes(false);
 		$active = $this->getServerConfigurationPrefixes(true);
 
-		if (!\is_array($all) || !\is_array($active)) {
-			throw new \Exception('Unexpected Return Value');
-		}
-
```

Comment:
This is correct - `getServerConfigurationPrefixes` always returns an array (even if it is empty). So this Exception cannot happen.